dialect/sql/schema/schema.go

```diff
@@ -576,3 +583,63 @@ func indexType(idx *Index, d string) (string, bool) {
 	}
 	return "", false
 }
+
+type driver struct {
+	sqlDialect
+	schema.Differ
+	migrate.PlanApplier
+}
+
+var drivers = func(v string) map[string]driver {
+	return map[string]driver{
+		entdialect.SQLite: {
+			&SQLite{WithForeignKeys: true},
+			sqlite.DefaultDiff,
+			sqlite.DefaultPlan,
+		},
+		entdialect.MySQL: {
+			&MySQL{version: v},
+			mysql.DefaultDiff,
+			mysql.DefaultPlan,
+		},
+		entdialect.Postgres: {
+			&Postgres{version: v},
+			mysql.DefaultDiff,
+			mysql.DefaultPlan,
+		},
+	}
+}
+
+// Dump the schema DDL for the given tables.
+func Dump(ctx context.Context, dialect, version string, tables []*Table, opts ...migrate.PlanOption) (string, error) {
```

Comment:
`Bootstrap` or sth-else, mean create the database from scratch

Reply:
It doesn't create anything. It simply dumps the schema.

Reply:
But the schema (SQL) generated here can be use to create the database, right?

Reply:
It can be used to create tables inside the schema. But they are not ordered for example (since it uses `PlanModeDump`)